app/gui/controller/engine-protocol/src/language_server.rs

```diff
@@ -177,6 +177,7 @@ trait API {
     , self_type   : Option<String>
     , return_type : Option<String>
     , tags        : Option<Vec<SuggestionEntryType>>
+    , is_static   : Option<bool>
```

Comment:
Shouldn't the attribute have a default value?

Reply:
These are the parameters of a search request. If you don't want to specify a parameter, you pass None